api/src/db/migrations/v97_updating_formation.test.ts

```diff
@@ -0,0 +1,259 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+import {
+  v96FormationData,
+  v96generateFormationAddress,
+  v96generateFormationFormData,
+  v96generatorProfileData,
+  v96UserDataGenerator,
+} from "./v96_added_date_field_to_tax_filing_data";
+import { migrate_v96_to_v97 } from "./v97_updating_formation_types";
+
+describe("migrate_v96_to_v97", () => {
+  beforeEach(async () => {
+    jest.resetAllMocks();
+  });
+
+  it("updates formationFormData for a limited-liability-company", () => {
```

Comment:
you have HUGE code diffs here that are honestly really hard to read through. This files is called "updating formation" and the test name is called "updates formationFormData" and is like 50 lines long.

Part of the value of tests is "tests-as-documentation" so that someone can read them and understand the behavior of the thing under test.

The way this is written, I have NO IDEA what changes have been made to formationFormData. 

One way to rewrite this in readable way would be to make use of a describe and beforeEach, for example. Something like:

describe: updating formationFormData for LLC
beforeEach: do the whole thing generate v96 and convert it to v97
it "changes x field in y way": make one assertion about one new behavior
it "changes q field in z way": same thing

Just have an it block with an assertion in it for every change. That way the tests would actually be describing what pieces of this massive object have changes to them